CHANGELOG.md

```diff
@@ -23,3 +30,8 @@ All notable changes to this project will be documented in this file
 
 ### Changed
 - Updated ruby version from `2.4.2` to `2.6.1`
+
+[2.0.1]: https://github.com/Invoca/boolean_expression/compare/v2.0.0...v2.0.1
+[2.0.0]: https://github.com/Invoca/boolean_expression/compare/v1.0.2...v2.0.0
+[1.0.2]: https://github.com/Invoca/boolean_expression/compare/v1.0.1...v1.0.2
+[1.0.1]: https://github.com/Invoca/boolean_expression/compare/v1.0.0...v1.0.1
```

Comment:
These all reference `boolean_expression`

Reply:
Nice catch.

Reply:
Also another thing I didn't realize was these tags haven't been made, so it'd be good to make those to prevent dead links, it should be relatively straightforward to find which commits they are relative to the PR's merged
```
v1.0.2
v1.0.1
v1.0.0
```

Reply:
Done